src/pathresolver.ts

```diff
@@ -28,21 +28,21 @@ class PathResolver {
   constructor(public abspath?: boolean) {
   }
 
-  isTemplatedUrl(href) {
+  isTemplatedUrl(href: string): boolean {
```

Comment:
PathResolver is going to undergo some refactoring.  I only added type annotations here to existing stuff because lint warnings.  Will add comments when refactored to support new root+abspath behavior.